docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_Faster_RCNN.md

```diff
@@ -1,10 +1,11 @@
-# Convert ONNX* Faster R-CNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Faster_RCNN}
+# Convert ONNX Faster R-CNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Faster_RCNN}
 
-These instructions are applicable only to the Faster R-CNN model converted to the ONNX* file format from the [facebookresearch/maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark).
+Below instructions are applicable **only** to the Faster R-CNN model converted to the ONNX file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark):
 
-**Step 1**. Download the pre-trained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn) (commit-SHA: 8883e49e68de7b43e263d56b9ed156dfa1e03117).
+1. Download the pre-trained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn):
+   * (commit-SHA: 8883e49e68de7b43e263d56b9ed156dfa1e03117).
 
-**Step 2**. To generate the Intermediate Representation (IR) of the model, change your current working directory to the Model Optimizer installation directory and run the Model Optimizer with the following parameters:
+2. Generate the models Intermediate Representation, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:
```

Comment:
```suggestion
2. Generate the model Intermediate Representation, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:
```

or

```suggestion
2. Generate the Intermediate Representation of the model, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:
```

Inanimate objects do not take possessive. It's better to use an adjective or "of" phrase.
I second the latter.